vector/src/main/res/layout/item_share_qr_code.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we use `item_` file prefix for Views we used as Epoxy items (i.e. list items). In your case, it is used for a dialog, so please prefix with `dialog_` so for instance `dialog_share_qr_code.xml`

Reply:
done